vllm/core/scheduler.py

```diff
@@ -1306,11 +1331,10 @@ def _allocate_and_set_running(self, seq_group: SequenceGroup) -> None:
         for seq in seq_group.get_seqs(status=SequenceStatus.WAITING):
             seq.status = SequenceStatus.RUNNING
 
-    def _append_slots(
-        self,
-        seq_group: SequenceGroup,
-        blocks_to_copy: List[Tuple[int, int]],
-    ) -> None:
+    def _append_slots(self,
+                      seq_group: SequenceGroup,
+                      blocks_to_copy: List[Tuple[int, int]],
+                      chunking_enabled: bool = False) -> None:
```

Comment:
Make them consistent.
```suggestion
                      enable_chunking: bool = False) -> None:
```